core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiServiceSpec.scala

```diff
@@ -829,13 +831,14 @@ class MethodConfigApiServiceSpec extends ApiServiceSpec with TestDriverComponent
   }
 
   it should "return 400 on copy Dockstore method configuration to Agora" in withTestDataApiServices { services =>
-    Post(copyToMethodRepo, httpJson(MethodRepoConfigurationExport("mcns", "mcn", testData.dockstoreMethodConfigName))) ~>
-      sealRoute(services.methodConfigRoutes) ~>
+    Post(s"/api${copyToMethodRepo}", httpJson(MethodRepoConfigurationExport("mcns", "mcn", testData.dockstoreMethodConfigName))) ~>
+      services.route ~>
```

Comment:
exceptionHandler is only wrapped around `services.route`, which is the actual route used in `Boot`...I'm actually not sure how the test was working fine before